lesson07/assignment/my-title/src/components/home.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <div>
+    <h1 class="display-1">{{ header }}</h1>
+      <div class="jumbotron" v-if="showJumbotron">
```

Comment:
One thing I'd suggest is looking at implementing other Bootstrap Vue components. Replacement for jumbotron is <b-jumbotron>. See documentation for all Bootstrap-Vue elements: [https://bootstrap-vue.org/docs/components/jumbotron](https://bootstrap-vue.org/docs/components/jumbotron)